frontend/templates/product/hooks/useProductReviews.ts

```diff
@@ -1,22 +0,0 @@
-import { useIFixitApiClient } from '@ifixit/ifixit-api-client';
-import { fetchProductReviews, Product } from '@models/product';
-import { useQuery } from 'react-query';
-
-const productReviewsKeys = {
-   reviews(productId: string) {
-      return ['product-reviews', productId];
-   },
-};
-
-export function useProductReviews(product: Product) {
```

Comment:
Why did we remove this usage of react-query? I'm pretty sure this is the exact use-case for it so we only end up querying the API one time instead of one time per component the data is is used in.

Reply:
yeah i think i got into my head this wasn't necessary because we weren't using initial data anymore